conn/physic/units.go

```diff
@@ -773,6 +817,12 @@ const maxInt64 = (1<<63 - 1)
 
 var maxUint64Str = "9223372036854775807"
 
+var (
+	errOverflowsInt64  = errors.New("exceeds maximum")
+	errUnderflowsInt64 = errors.New("exceeds minimum")
```

Comment:
That's OverflowNegative or something like that.
Since this is not exported, it's not a big deal here.

I think it'd be nice for underflow to actually return an error, but that's good for a follow up, since this change dragged on long enough.